File: dataset/human/c/file_2597.c
#include <stdio.h>
int main(void)
{
  char p,q,r;
  int a=0,b=0,c=0;
  scanf("%s%s%s",&p,&q,&r);
  switch(p)
  {
    case 'a':
      a++;
      break;
    case 'b':
      b++;
      break;
    case 'c':
      c++;
      break;
  }
  switch(q)
  {
    case 'a':
      a++;
      break;
    case 'b':
      b++;
      break;
    case 'c':
      c++;
      break;
  }
  switch(r)
  {
    case 'a':
      a++;
      break;
    case 'b':
      b++;
      break;
    case 'c':
      c++;
      break;
  }
  if (a==1&&b==1&&c==1) printf("Yes\n");
  else printf("No\n");
}

File: dataset/human/java/file_2597.java
import java.util.*;
public class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		String s = sc.next();
		String t = sc.next();
		char[] s2 = s.toCharArray();
		char[] t2 = t.toCharArray();
		Arrays.sort(s2);
		Arrays.sort(t2);
		t2[0]=t2[t2.length-1];
		String S = String.valueOf(s2);
		String T = String.valueOf(t2);
		if(S.compareTo(T)<0) {
			System.out.println("Yes");
		}else {
			System.out.println("No");
		}
	}
 
}
